src/styles/sidebar/components/annotation-publish-control.scss

```diff
@@ -0,0 +1,119 @@
+@import './primary-action-btn';
+
+// A composite control consisting of an `annotation-publish-btn` and a
+// cancel button
+.annotation-publish-control {
+  display: flex;
+
+  &__cancel-btn {
+    @extend .btn--cancel;
+
+    margin-left: 5px;
+    font-weight: normal;
+
+    &__icon {
+      margin-right: 3px;
+      transform: translateY(10%);
+    }
+  }
+
+  // A split button with a primary submit on the left and a drop-down menu
+  // of related options to the right
+  .annotation-publish-control__btn {
+    $text-color: $color-seashell;
+    $default-background-color: $color-dove-gray;
+    $hover-background-color: $color-mine-shaft;
```

Comment:
I guess these colors were taken from the existing implementation? That's OK, although something we should do is replace our three slightly-different color palettes with the latest one that is shared with h/lms (those have names `$grey-{1,2,3,4,5,6,7}`).